pyndustri.pyi

```diff
@@ -413,14 +413,31 @@ class Unit(Senseable):
 
         found, x = Unit.locate('enemy', building='core')  # the x coordinate of the enemy core if found
         found, x, y = Unit.locate('enemy', building='core')  # the x and y coordinate of the enemy core if found
-        found, x, y, building = Unit.locate('enemy', building='core')  # the x and y coordinate of the enemy core if found, and specific building type
+        found, x, y, building = Unit.locate('enemy', building='core')  # the x and y coordinate of the enemy core if found, and specific building
```

Comment:
It returns building object whose properties can be accessed via sensor. It does evaluate to type when trying to print it, hence why I believe it was written as such